python/paddle/v2/framework/tests/test_elementwise_mod_op.py

```diff
@@ -0,0 +1,36 @@
+import unittest
```

Comment:
should remove this file since or put under `v2/fluid/test`

Reply:
I have another PR in of https://github.com/PaddlePaddle/Paddle/pull/5454. which I can move this file to the correct place in that PR.  
This PR is so huge and fragile, I think we can mark that and do it later.

Reply:
OK